assets/sass/modules/_forgot-password.scss

```diff
@@ -53,10 +62,6 @@
     padding-bottom: 15px;
   }
 
-  a.button {
```

Comment:
i hope removing this won't cause any other alignment issues ?

Reply:
It's not needed anymore